Makefile.include

```diff
@@ -1,3 +1,10 @@
+MATCH_MAKE_VERSION = 5.%
```

Comment:
__PLEASE NOTE__ _The version here is 5 just to show what happens if you have an old version, so it can be tested._

Without the test commit, the required version is 4.